pkg/controller/batchrelease/workloads/workload_controller_types.go

```diff
@@ -30,6 +30,7 @@ const (
 	WorkloadPodTemplateChanged WorkloadEventType = "workload-pod-template-changed"
 	WorkloadReplicasChanged    WorkloadEventType = "workload-replicas-changed"
 	WorkloadStillReconciling   WorkloadEventType = "workload-is-reconciling"
+	WorkloadHasGone            WorkloadEventType = "workload-has-gone"
```

Comment:
plz comment about every workload event type, especially for event type like IgnoreWorkloadEvent, it is unclear when such event will occur and what should be done for such event.